browser_extension/js/main.js

```diff
@@ -16,6 +16,10 @@ $(function () {
   var $loginPage = $('.login.page'); // The login page
   var $chatPage = $('.chat.page'); // The chatroom page
   var $googleSignInButton = $('#btnGoogleSignIn'); // Google login button
+  var $guestSignInButton = $('#btnGuestLogin'); //Guest login button
+  var $guestSignInCheck = $('#btnGuestCheck'); //Guest login button
```

Comment:
What is the purpose of the `$guestSignInCheck`. I'm not sure when it's being used.